### PR TITLE
WT-6882 Files created during incremental backup should be copied in full.

### PR DESCRIPTION
**Pull request by sueloverso**
@keitharnoldsmith the change turned out to be a lot smaller than expected. I will open a separate ticket to modify the test_incr_backup to vary the allocation size. There have been a handful (7-8) failures noted in the ticket. I have run this change on the seeds from all failures successfully with this change.

### CODE REVIEW

**Review by keitharnoldsmith — APPROVED** [review]
Nice!  Glad it turned out to be simpler than expected.

LGTM!

**Comment by sueloverso** [comment]
@keitharnoldsmith I reverted the original change and took a slightly different approach to handle across a reopen. @ddanderson, you can read the analysis in the ticket about what is root cause. I added variance for allocation size and closing/reopening the connection to `incr_backup`. I reran all 6 failing seeds on this branch and they pass.

**Comment by sueloverso** [comment]
I have also kicked off a script to run randomized `test_incr_backup` in a loop and will let that run for several hours.

File: src/include/os.h [hunk 1]
```diff
@@ -111,6 +111,7 @@ struct __wt_fh {
     TAILQ_ENTRY(__wt_fh) hashq;     /* internal hash queue */
     u_int ref;                      /* reference count */
     WT_FS_OPEN_FILE_TYPE file_type; /* file type */
+    bool created_during_bkup;       /* file created during incremental backup */
```
Comment (ddanderson) [inline src/include/os.h:114]: Looks like this field is unused.

**Review by ddanderson — APPROVED** [review]
LGTM - one small cleanup.

**Review by keitharnoldsmith — APPROVED** [review]
LGTM!

File: src/include/os.h (hunk 1 above)
Comment (sueloverso) [reply src/include/os.h]: Good catch. That was from an earlier iteration/attempt. Fixed.

**Pull request merged**